server/protocols/webadmin-cli/src/main/java/org/apache/james/cli/mailbox/MailboxCreateCommand.java

```diff
@@ -52,13 +52,11 @@ public Integer call() {
             if (rs.status() == CREATED_CODE) {
                 mailboxCommand.out.println("The mailbox was created successfully.");
                 return WebAdminCli.CLI_FINISHED_SUCCEED;
-            } else if (rs.status() == INVALID_MAILBOX_NAME_CODE) {
-                mailboxCommand.out.println("Invalid mailbox name.\n" +
-                    "Resource name mailboxNameToBeCreated should not be empty, nor contain # & % * characters.");
+            } else if (rs.status() == BAD_REQUEST_CODE) {
+                mailboxCommand.err.println(rs.body());
                 return WebAdminCli.CLI_FINISHED_FAILED;
             } else if (rs.status() == USERNAME_NOT_EXIST_CODE) {
-                mailboxCommand.out.println("The user name does not exist.\n" +
-                    "Resource name usernameToBeUsed should be an existing user.");
+                mailboxCommand.err.println(rs.body());
```

Comment:
really? we just print out 404? The user is supposed to understand what from this? (idem for above)

Reply:
Ignore this one, I got it...